app/main.py

```diff
@@ -0,0 +1,143 @@
+import json
+import os
+import sys
+
+from pathlib import Path
+
+import streamlit as st
+
+if os.getcwd() not in sys.path:
+    sys.path.append(os.getcwd())
```

Comment:
Just use this command to run your app, it will handle all sub packages automatically.
```shell
python -m streamlit run app/main.py
```

Reply:
Ok, it works. Removed them!